web/settings.py

```diff
@@ -149,10 +149,14 @@ class Dev(Common):
 
     INTERNAL_IPS = ["127.0.0.1"]
 
+    ENABLE_NEW_RELIC = False
+
 
 class Prod(Common):
     SECRET_KEY = values.SecretValue()
     ALLOWED_HOSTS = values.ListValue()
     DATABASES = {"default": dj_database_url.config(conn_max_age=600, ssl_require=True)}
     GOOGLE_ANALYTICS_KEY = values.Value()
     CITY_COUNCIL_WEBSERVICE = values.Value()
+    ENABLE_NEW_RELIC = True
+    NEW_RELIC_CONFIG_FILE = os.getenv("NEW_RELIC_CONFIG_FILE", "newrelic.ini")
```

Comment:
Tem alguma chance de termos arquivos de configuração diferentes para o new relic?

Reply:
@anapaulagomes você diz um arquivo novo tipo web/newrelic_settings.py ?

Reply:
Não, de ter um outro que não seja `newrelic.ini`. Se não, a minha sugestão seria deixar o valor hardcoded mesmo.

```suggestion
    NEW_RELIC_CONFIG_FILE = "newrelic.ini"
```

Reply:
Não acho que tenha mesmo, mas sempre gosto de oferecer a opção de mudança de parâmetros, se vc prefere o nome hardcoded, eu estou ok com isso ;)

Reply:
Deixamos como está então :)

Reply:
IMHO pelo que entendi da documentação do NewRelic sim, eu posso dar outro nome ao arquivo newrelic.ini, por isso eu defino qual é o arquivo pela variável de ambiente `NEW_RELIC_CONFIG_FILE`, pois se não fosse isso não vejo muito sentido ser necessário definir a variável de ambiente antes de executar o serviço do newrelic. Faz sentido isso para vocês?